woodwork/tests/logical_types/test_logical_types.py

```diff
@@ -110,6 +111,14 @@ def test_datetime_transform(datetimes):
         assert str(transform.dtype) == 'datetime64[ns]'
 
 
+def test_datetime_conversion_error():
+    series = pd.Series(['a', 'b', 'c'])
+    match = 'Error converting datatype for None from type object to type datetime64[ns]. '
```

Comment:
Minor detail, but I think it would be good to name the series in this test just to make sure that the series name is being displayed properly.